### ISSUE TITLE
Need "restart at" function

### ISSUE THREAD

**Issue by pos-ei-don** [ScratMan/HASmartThermostat#272]
I have a very powerful gas heating device  and a use smart Thermostate for the mixing-pumps.
I use this.
https://www.ebay.de/itm/124225295470?chn=ps&_ul=DE&norover=1&mkevt=1&mkrid=707-166974-037691-2&mkcid=2&mkscid=101&itemid=124225295470&targetid=2348818482861&device=m&mktype=pla&googleloc=1000953&poi=&campaignid=19896375258&mkgroupid=166806809739&rlsatarget=pla-2348818482861&abcId=9309148&merchantid=115521014&geoid=1000953&gad_source=1&gclid=CjwKCAiAgoq7BhBxEiwAVcW0LLN5kxHR4Tx0Qc65JGrsFMxcBxWe7EH-IQu7bZpIrEry0xLIoYPXEBoCuowQAvD_BwE

IT works great, but when the Gas-Theater starts, I know that I have to close the valve to 18%, otherwise the wood en floor gets massive overheated. (wood is not allowed to get more tha ln 28°C).

I need a possibillity to set the PID to restart at 18%.

Please, help me.

**Comment by ScratMan** [comment]
I'm not sure to understand. The thermostat controls the water temperature at the output of the mixing valves?
If yes, there is no need for a limit, but you need very fast sampling rate on the temperature sensors.

But normally, a mixing valve for floor heating is already a temperature regulator, so I don't understand how you control it with the smart thermostat.

**Comment by pos-ei-don** [comment]
Yes, I need a very very fast Regulation in the Event when the Gas burner starts. This is about an hour a Day.
The Rest of the Day I need a very slow Regulation, as the temperature only Changes slow, and, most important, the Mixer only takes about 30-90 Changes a Day.
This is the Mixer of the hole string, this is connected to the 20  floor heatings in its rooms. 

The Mixer is like a Cover or valve. I can send it an value between 0-100. It does not have its own temp sensor. 
It needs about 210s of mfull Travel time, so i cannot Update the value often or within this 210s.


I know, that a value of 18% is closed enough that the heat rush of the Gas burner does not hurt.

So I really need a possibiluty to restart the pid with a give opening. From there on it can regulate as usual, nö Changes needed. Close a bit if too hot, open a bit it too cold.
